build.sbt

```diff
@@ -1,10 +1,14 @@
-Global / onChangedBuildSource := ReloadOnSourceChanges
+val scala3 = "3.1.0"
+val scala213 = "2.13.4"
+val scala212 = "2.12.13"
```

Comment:
Maybe upgrading to the newest would help? 3.1.1, 2.13.8 and 2.12.15

Reply:
It was a bug a Scala Native unrelated with the Scala version.